packages/opentelemetry-plugin-postgres/opentelemetry-plugin-pg-pool/src/enums.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum AttributeNames {
+  COMPONENT = 'component',
+}
```

Comment:
Are any of the attributes from [pg](https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-plugin-postgres/opentelemetry-plugin-pg/src/enums.ts) applicable here?

Reply:
yes, will fix this.